lib/dictionary

```diff
@@ -0,0 +1,168 @@
+personal_ws-1.1 en 0
+actionables
+AdVenture
+AIME
+AMC
+api
+API
+APIs
+Arduino
+bannings
+Bloomberg
+bogidon
+bolded
+cancelled
+CAPTCHA
+checkbox
+checkboxes
+Christabel
+Chromebook
+Chromebooks
+Codecademy
+CodeHS
+could've
+cringiest
+cringy
+CSS
+CTF
+CTFs
+Ctrl
+CVV
+DDoS
+de
+decrement
+Django
+Dropbox
+EasyCTF
+EasyCTF's
+Flappy
+flyer
+flyers
+FreeBSD
+freenode
+Fu
+fundraise
+Gaberiel
+GIF
+GIFs
+girlsWhoCode
```

Comment:
I believe it's the name of a club at @saescapa's high school, I got tripped up on this as well.